SPECS/containerized-data-importer/cdi.spec

```diff
@@ -103,7 +103,7 @@ kubernetes installation with kubectl apply.
 # Note: having bar symlink'ed to DIR/src/foo/bar does not seem to work. Looks
 # like symlinks in go path are not resolved correctly. Hence the sources need
 # to be 'physically' placed into the proper location.
-%setup -q -n go/src/kubevirt.io/%{name} -c -T
+%setup -q -n go/src/kubevirt.io/containerized-data-importer -c -T
```

Comment:
Can we use autosetup here instead?

Reply:
> Can we use autosetup here instead?

As I read https://rpm-software-management.github.io/rpm/manual/autosetup.html autosetup helps simplification of patches.  Are you suggesting to ready the framework as part of this PR?

Reply:
I think a representative example would be: [m2crypto](https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS/m2crypto/m2crypto.spec#L41)

The benefit being, once we try to fix the CVE(s) down the line, it becomes trivial as we only include `PatchN: CVE-XYZ-ABC`. We do not have to explicity apply the patch using `patchN -p1` :)

Reply:
Replaced %setup with %autosetup